internal/pkg/pluginmanager/pluginmd5_test.go

```diff
@@ -0,0 +1,83 @@
+package pluginmanager
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/merico-dev/stream/internal/pkg/configloader"
+	md5helper "github.com/merico-dev/stream/pkg/util/md5"
+)
+
+// TestCheckLocalPlugins test plguin .so matches dtm core md5
+func TestCheckLocalPlugins(t *testing.T) {
```

Comment:
I think this is more of a functional test, not unit test. For functional tests, we are already using the ginkgo framework. Please use ginkgo. If it's only unit tests, Go's "testing" is fine.

Reply:
It's also good to add unit test for functions like checkPluginMismatch.

Reply:
I think CheckLocalPlugins(config) is also a function, just like others

Reply:
I think the test for "FileMatchesMD5" is a "unit" test (technically speaking, it isn't, since it has dependencies on the os.Create, but at least the function is relatively isolated without other dependencies).

The test for CheckLocalPlugins, on the other hand, depends on creating a config, calling configloader's functions to prepare for the test, etc., which are some external dependencies. Yes, technically speaking, the test for "CheckLocalPlugins" isn't "functional" test either, but given the complexity of building the dependencies and stuff, I think we would benefit from using ginkgo.